### PR TITLE
Table: fix chrome crash when set thead css display to none issue (#16427)

### LINKED ISSUE #16427: [Bug Report] Chrome will crash when set Table Component 'thead' css 'display' to none
### Element UI version
2.10.1

### OS/Browsers version
MacOS 10.13.6 / Google Chrome Version 75.0.3770.100 (Official Build) (64-bit)

### Vue version
2.6.10

### Reproduction Link
https://codepen.io/pen >> https://element.eleme.io/#/zh-CN/component/table

### Steps to reproduce
1. Open chrome dev tools
2. Find any element of thead.
3. Add 'display:none' rule for the '.el-table thead' selector.
4. Then Chrome crashed.

PS. The issue link rule is bypassed, because links on official websites can reproduce this problem very well.
The reproduction link is https://element.eleme.io/#/zh-CN/component/table

### What is Expected?
Chrome won't crash,and the table header can be hidden.

### What is actually happening?
Chrome crashed.

### PR DESCRIPTION
**Pull request by luckyCao**
fix #16427 
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: packages/table/src/table-layout.js [hunk 1]
```diff
@@ -113,6 +117,17 @@ class TableLayout {
     this.notifyObservers('scrollable');
   }
 
+  headerDisplayNone(elm) {
```
Comment (ziyoung) [inline packages/table/src/table-layout.js]: 这里的判断也许不用那么麻烦。

屏幕快照 2019-08-08 下午7 50 17

即便是 header 没有内容，但是高度依然存在。或者检测高度（如果 header 显示，高度一定大于 0）是否存在也是一种思路。

Comment (luckyCao) [reply packages/table/src/table-layout.js]: 检测高度不行的吧 下面有拿高度判断的

**Comment by asself** [comment]
什么时候解决呀

**Comment by lzw621558** [comment]
vue.runtime.esm.js?2b0e:619 [Vue warn]: Error in nextTick: "TypeError: Cannot read property 'querySelector' of undefined"

vue.runtime.esm.js?2b0e:1888 TypeError: Cannot read property 'querySelector' of undefined
    at TableLayout.updateElsHeight (element-ui.common.js?5c96:11369)
    at TableLayout.setHeight (element-ui.common.js?5c96:11328)
    at eval (element-ui.common.js?5c96:11323)
    at Array.eval (vue.runtime.esm.js?2b0e:1980)
    at flushCallbacks (vue.runtime.esm.js?2b0e:1906)
看了下源码，表格这个地方设置show-header = false后，
var headerTrElm = headerWrapper.querySelector('.el-table__header tr');
然后发现没有这个元素，就报了上边的错误。
希望尽快修复0——0

**Pull request merged**